net/isc-dhcp/files/dhcpd.init

```diff
@@ -377,15 +462,23 @@ static_host_add() {
 	for ip in $ips; do
 		revip="$(rev_str "$ip" ".")"
 
-		update "$name.$domain." IN A "$ip"
-		update "$revip.in-addr.arpa." IN PTR "$name.$domain."
+		update "$name.$domain." "$leasetime" IN A "$ip"
```

Comment:
I guess since these are the only two RR's associated with a lease, it makes senses that they're bounded by the lease time...

Reply:
@jempatel ?